mitodl/bootcamp-ecommerce
# fix: missing migration and migrations detection scripts

#### What are the relevant tickets?
None, But it was noticed during testing. It's a followup on https://github.com/mitodl/bootcamp-ecommerce/pull/1507, The migration was missing from that PR.

#### What's this PR do?
- Adds a missing migration
- Fixes why the scripts didn't detect that
- Ports over the xPRO migration detection scripts because they don't use a temporary file.

#### How should this be manually tested?
- Check the tests are passing
- If you want to do more testing, Try testing the auto & missing migration scripts locally

#### Where should the reviewer start?
(Optional)

#### Any background context you want to provide?
This was left in https://github.com/mitodl/bootcamp-ecommerce/pull/1507
